### PR TITLE
LL-983 Sentry reports, link cleanups

### PR DESCRIPTION
**Pull request by juan-cortes**
Added more reported errors to Sentry, removed unneeded catch blocks for links.

### CODE REVIEW

File: src/libcore/scanAccountsOnDevice.js [hunk 1]
```diff
@@ -93,7 +94,7 @@ export const scanAccountsOnDevice = (
         o.complete();
       } catch (e) {
         const mappedError = remapLibcoreErrors(e);
-
+        Sentry.captureException(mappedError);
```
Comment (gre) [inline src/libcore/scanAccountsOnDevice.js]: prefer passing the `e` that is the raw error

File: src/screens/SendFunds/06-Validation.js [hunk 2]
```diff
@@ -109,6 +110,8 @@ class Validation extends Component<Props, State> {
           if (e && e.statusCode === 0x6985) {
             error = new UserRefusedOnDevice();
           }
+
+          Sentry.captureException(error);
```
Comment (gre) [inline src/screens/SendFunds/06-Validation.js]: it should be in the `else` case of the if above. `UserRefusedOnDevice ` is legit

**Comment by gre** [comment]
all good! some minor changes request.

File: src/libcore/scanAccountsOnDevice.js (hunk 1 above)
Comment (juan-cortes) [reply src/libcore/scanAccountsOnDevice.js]: On it!

**Comment by juan-cortes** [comment]
Done @gre

**Pull request merged**